hbp_knowledge/oxidative-stress/shefa2019.bel

```diff
@@ -74,114 +74,147 @@ DEFINE ANNOTATION MeSHDisease   AS URL "https://arty.scai.fraunhofer.de/artifact
 role against mitochondrial dysfunction and apoptosis with
 mitochondrial quality control by activating PINK1/PAR-
 KIN-induced mitophagy (Figure 2) (Fivenson et al., 2017). "
+SET Confidence = "Medium"
 p(HGNC:PINK1) -| path(HBP:"mitochondrial dysfunction")
 p(HGNC:PINK1) -| bp(GO:"apoptotic process")
-p(HGNC:PINK1) -> bp(GO:mitophagy)
+p(HGNC:PINK1) -> act(p(HGNC:PRKN))
+act(p(HGNC:PRKN)) -> bp(GO:mitophagy)
+UNSET Confidence
 
 SET Evidence = "The role of PINK1 has been
 well-documented in neurodegenerative and aging-related
 diseases (Li and Hu, 2015). "
+SET Confidence = "High"
 p(HGNC:PINK1) -- path(MESH:"Neurodegenerative Diseases")
+UNSET Confidence
 
 SET Evidence = "The cytosolic E3 ubiquitin ligase PARKIN and mitochon -
 drial PINK1 have been implicated in the abnormal expres-
 sion of genes associated with a recessive form of Parkinson-
 ism (Schiavi and Ventura, 2014). "
+SET Confidence = "High"
 p(HGNC:PRKN) -- path(MESH:"Parkinson Disease")
 SET MeSHAnatomy = "Mitochondria"
 p(HGNC:PINK1) -- path(MESH:"Parkinson Disease")
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "PINK1 is then degrad-
 ed by various proteases, such as mitochondria-processing
 protease (MPP + ), the inner membrane presenilin-related
 rhomboid-like protease (Meissner et al., 2011)."
 #MPP+ is complex(p(HGNC:PMPCA),p(HGNC:PMPCB))
+SET Confidence = "High"
 complex(p(HGNC:PMPCA),p(HGNC:PMPCB)) => deg(p(HGNC:PINK1))
+UNSET Confidence
 
 SET Evidence = " Following
 mitochondrial depolarization, PINK1 is translocated to the
 inner mitochondrial membrane, degraded, and sustained on
 the mitochondrial membrane (Lazarou et al., 2012)."
+SET Confidence = "High"
 bp(GO:"mitochondrial depolarization") -> p(HGNC:PINK1, loc(GO:"mitochondrial inner membrane"))
 p(HGNC:PINK1, loc(GO:"mitochondrial inner membrane")) -> deg(p(HGNC:PINK1))
+UNSET Confidence
 
 SET Evidence = "The ag -
 gregation of PINK on the mitochondrial surface induces mi-
 tophagy by volunteering PARKIN to degrade mitochondria
 via a mechanism that is not completely understood. "
+SET Confidence = "High"
 p(HGNC:PINK1, loc(GO:"mitochondrial outer membrane")) -> act(p(HGNC:PRKN))
 act(p(HGNC:PRKN)) -> deg(a(MESH:Mitochondria))
+UNSET Confidence
 
 SET Evidence = "Translocation of PARKIN to damaged mitochondria has
 been shown to weaken PINK1 function (Geisler et al., 2010)."
+SET Confidence = "High"
 p(HGNC:PRKN, loc(MESH:Mitochondria)) -| act(p(HGNC:PINK1))
+UNSET Confidence
 
 SET Evidence = "The mitochondrial fusion proteins
 mitofusion 1 (Mfn1) and mitofusion 2 (Mfn2) have been
 recognized as substrates of PARKIN (Rakovic et al., 2011),
 as illustrated in Figure 2. "
-p(HGNC:PRKN) -> p(HGNC:MFN1, pmod(Ub))
-p(HGNC:PRKN) -> p(HGNC:MFN2, pmod(Ub))
+SET Confidence = "Medium"
+composite(p(HGNC:PRKN),p(HGNC:MFN1, pmod(Ub))) -> complex(p(HGNC:PRKN),p(HGNC:MFN1, pmod(Ub)))
+composite(p(HGNC:PRKN),p(HGNC:MFN2, pmod(Ub))) -> complex(p(HGNC:PRKN),p(HGNC:MFN2, pmod(Ub)))
+UNSET Confidence
```

Comment:
please keep line 183 and 184 and delete 215 and 216; afterwars confidence can be set on high